distributor/src/distributor/division_codes.bal

```diff
@@ -0,0 +1,53 @@
+// Map contains all the division codes
+map<string> divisionCodeMap = { "01A": "COLOMBO-NORTH", "01B": "COLOMBO-CENTRAL", "01C": "BORELLA", "01D":
```

Comment:
We can make this a `const` right?

Reply:
Shall we also reformat this to one field per line? :) VSCode formatting should fix it.